rackcity/views/asset_views.py

```diff
@@ -60,20 +62,6 @@
 from rackcity.models.asset import get_next_available_asset_number
 
 
-# @close_old_connections_decorator
-@api_view(['GET'])  # DEPRECATED !
```

Comment:
I deleted all deprecated function views & comments to clean up some of these view files